src/features/create_layer/CustomLayerDrawToolsWidget.tsx

```
@@ -0,0 +1,75 @@
+  const isMobile = useMediaQuery(IS_MOBILE_QUERY);
+
+  switch (state) {
+    case 'disabled':
+    case 'regular':
+      return null;
+
+    case 'active':
+      const onFinish = () => {
+        finishDrawing(); // order of callings is important!
+        onClick();
+      };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)